src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -70,18 +71,19 @@ def _submit_job(self, message: Message, timeout: Optional[float]) -> Message:
         # Register state
         self.proxy_state.register_context(run_id=run_id)
 
-        # Retrieve state
-        state = self.proxy_state.retrieve_context(run_id=run_id)
+        # Retrieve context
+        context = self.proxy_state.retrieve_context(run_id=run_id)
+        partition_id = str(context.partition_id)
```

Comment:
Should we rename this to indicate that it's not the typical `partition_id: int`?
```suggestion
        partition_id_str = str(context.partition_id)
```